lib/Console/Traits/Parallelization.php

```diff
@@ -33,6 +33,10 @@ trait Parallelization
 
     use ParallelizationBase;
 
+    /**
+     * @deprecated Deprecated since webmozarts/console-parallelization 2.0.0 and will be removed in webmozarts/console-parallelization 3.0.0. Use ParallelizationInput::configureCommand() instead.
```

Comment:
@mattamon Shouldn't we directly implement `configureCommand()`?

Reply:
Yes I thought about that too, but the configureCommand is not part of the  \Webmozarts\Console\Parallelization\Parallelization Trait anymore, it is now it's own class, so I am not sure how to handle this.
https://github.com/webmozarts/console-parallelization/blob/main/src/Input/ParallelizationInput.php

Reply:
@mattamon you can use that probably in the `configure()` method ...?

Reply:
I saw that comment too, but I am not sure about it since there is no configure method in the trait.

Reply:
The `configure()` method you implement in the command, and there you call the method from the trait to configure the command, see e.g.: 
https://github.com/pimcore/pimcore/blob/11.x/bundles/CoreBundle/src/Command/CacheClearCommand.php#L43